editor/editor_log.cpp

```diff
@@ -379,6 +380,13 @@ void EditorLog::_reset_message_counts() {
 	}
 }
 
+void EditorLog::_check_need_rebuild() {
+	if (need_rebuld && !unlikely(log->is_updating())) {
```

Comment:
```suggestion
	if (need_rebuld && !log->is_updating()) {
```
See previous comment, this should be unneeded here and the current syntax hurts readability